storybook/stories/API/shapes/Cross.mdx

```diff
@@ -3,14 +3,14 @@ import * as CrossStories from './Cross.stories';
 
 # Cross
 
-The Cross is a shape that itself is not a valid child in any component. 
+The Cross is a shape that itself is not a valid child in any component.
 It can be used to implement custom elements, returning it from within a
 
 \```jsx
```

Comment:
Let's remove this back slash too, I think it causes some confusion for the MDX compiler it things that the ArgTypes is not closed properly

```suggestion
```jsx
```

Reply:
that was because we were missing a `/` in `Customized` and didn't have anything to do with `ArgTypes` though the error was misleading. Will make the changes anyways

Reply:
It was my bad actually I didn't know you could escape triple backticks with ~~~ and had to add that symbol to make the suggestion look like what it was :D